src/main/java/org/stellar/sdk/requests/EventListener.java

```diff
@@ -9,4 +9,6 @@
    * @param object object deserialized from the event data
    */
   void onEvent(T object);
+
+  void onError();
```

Comment:
pass the exception as a parameter; probably a common use case for onError will be logging